scripts/chroot_reconst.sh

```diff
@@ -80,6 +80,9 @@ cat /usr/share/applications/lxqt-about.desktop
 sed -i '$ a NoDisplay=true' /usr/share/applications/lxqt-about.desktop
 cat /usr/share/applications/lxqt-about.desktop
 
+# Meilix default settings
+dpkg -i --force-overwrite meilix-default-settings_1.0_all.deb
```

Comment:
can you explain force overwrite why it is required ?

Reply:
Because some of the configuration files are already downloaded in the ISO, so we will have to overwrite them forcely for the custom ISO

Reply:
looks hackish default settings were installed before then some other files are added/replaced why overwrite them?

Reply:
No @xeon-zolt as mentioned in #360 @tabesin has made a new version of the chroot script and we are installing the meilix-default settings package for the first time